pkg/report/report.go

```diff
@@ -31,7 +31,7 @@ type Report struct {
 	Title string
 	// Report contains whole oops text.
 	Report []byte
-	// Output contains whole raw kernel output.
+	// Output contains whole raw console output as passed to Reporter.Parse().
```

Comment:
I hate to nitpick: but Go code never uses `()` after functions in comment and in phrase.

Reply:
Fixed.